napari/_qt/layer_controls/qt_shapes_controls.py

```diff
@@ -154,7 +168,7 @@ def __init__(self, layer):
             layer,
             'delete_shape',
             slot=self.layer.remove_selected,
-            tooltip='Delete selected shapes',
+            tooltip=f"Delete selected shapes ({KEY_SYMBOLS['Backspace']})",
```

Comment:
side-note;  we seem to often have the logic to use KEY_SYMBOL... would it be worth having a utils function somewhere that format shortcuts ? There are often other differences in rendering, like windows tend to separate keys with -, like `Ctrl-Alt-Del`, while mac will just stick the symbols w/o separator `⌘⌥⌫`.

We can likely do that as a separate PR if that sounds ok.

Reply:
yeah, that sounds like a fine thing for a follow up PR